### ISSUE TITLE
Homepage: "Dive into Python" code sample five: add Unicode animal(s)

### ISSUE THREAD

**Issue by jayaddison** [python/pythondotorg#2230]
**Is your feature request related to a problem? Please describe.**
The Python homepage includes a slideshow of five sample Python programs, to provide a sense of what code written in the language looks like, and how to perform some common activities.

[Code sample 5](https://github.com/python/pythondotorg/blob/b41f5e461f077be9d6999a3d42d9c23f8edcbaab/codesamples/factories.py#L27-L49) is about reading input from the user and printing output, and has the comment-title `# Simple output (with Unicode)`.

![image](https://user-images.githubusercontent.com/55152140/213232974-40f360cc-5b29-4448-9b0c-65a91552cd3e.png)

(pedantically speaking: the sample code shown _does already_ use Unicode, if the reader is using Python 3 - because Unicode is the native string format for that version)

It'd be nice (and fun) to include at least one non-typical Unicode character - like an emoji animal character - to demonstrate the extent of Unicode support within Python more clearly.

**Describe the solution you'd like**
It would be great if code sample 5 of the "Dive into Python" section on the Python homepage could include at least one Unicode animal.

Perhaps the conversation could be updated to demonstrate a conversation between a snake and a robot?  Are robots and snakes compatible as friends?  I'm not really sure.

**Describe alternatives you've considered**
- Python speaks to Python (small diff, but potentially not that interesting)
- Python speaks to a bird (could be risky)
- Python speaks to a different kind of snake (would be nice, although I don't know whether Unicode provides symbols representing that many different kinds of snake)

**Additional context**
The tricky part may be updating the [HTML representation of the code sample's output](https://github.com/python/pythondotorg/blob/b41f5e461f077be9d6999a3d42d9c23f8edcbaab/codesamples/factories.py#L31) that appears on the homepage.  Although HTML does support Unicode, reference materials may be helpful to properly encode some characters.

**Comment by jayaddison** [comment]
(note: I'm unsubscribing from this issue on GitHub since I don't think I need to be involved for the implementation, and because it's not urgent)

**Comment by jayaddison** [comment]
@JacobCoffee I don't think that the latest changes to the code samples implemented/resolved this -- and that's OK!  it's a trivial request -- but please could you re-close this issue as wontfix/not-planned, or re-open it?  Thank you either way.

**Comment by jayaddison** [comment]
Thanks 👍

**Comment by JacobCoffee** [comment]
Will keep this in mind for an eventual Python.org rewrite although ideally for me we'd just move to a Playground with examples there in some WASM runtime
